packages/react-components/react-menu/src/components/MenuItem/useMenuItemStyles.styles.ts

```diff
@@ -27,6 +27,7 @@ const useStyles = makeStyles({
     paddingRight: '10px',
     paddingLeft: '10px',
     height: '32px',
+    minHeight: '32px',
```

Comment:
why is the `minHeight` necessary?

Reply:
To prevent menu item shrink, without it, on small viewport it looks like:
<img width="199" alt="image" src="https://github.com/microsoft/fluentui/assets/28751745/c2f2477d-ffcf-4e83-8794-02a1a12231cf">